config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -100,7 +100,7 @@ periodics:
       - name: PARALLEL
         value: "true"
       - name: FEATURE_GATES
-        value: '{"AllBeta":true}'
+        value: '{"AllBeta":true, "MultiCIDRServiceAllocator":false}'
```

Comment:
We literally pass `AllBeta=true` to the feature-gates flag, it's not getting re-ified into individual gates.
I don't think this will fix it.